include/aca_on_demand_engine.h

```diff
@@ -17,22 +17,39 @@
 
 #define IPTOS_PREC_INTERNETCONTROL 0xc0
 #define DSCP_DEFAULT (IPTOS_PREC_INTERNETCONTROL >> 2)
-#define STDOUT_FILENO   1   /* Standard output.  */
+#define STDOUT_FILENO 1 /* Standard output.  */
 
 #include "common.pb.h"
 #include <openvswitch/ofp-errors.h>
 #include <openvswitch/ofp-packet.h>
 #include <string>
+#include <thread>
+#include "hashmap/HashMap.h"
+#include <grpcpp/grpcpp.h>
+#include <unordered_map>
 
 using namespace alcor::schema;
-
+using namespace std;
+// using namespace grpc;
+struct data_for_on_demand_call {
+  //     alcor::schema::OperationStatus on_demand_reply;
+  string uuid;
+  uint32_t in_port;
+  void *packet;
+  int packet_size;
+  alcor::schema::Protocol protocol;
+};
```

Comment:
@zzxgzgz the name for a structure quite a long, is it better replace `data_for_on_demand_call` with `on_demand_payload`?

Reply:
fixed